owa/app/css/openmrs-owa-idgen.css

```diff
@@ -17,7 +17,6 @@ body {
     font-size: 16px;
 }
 
-
 nav img {
     width: 120px !important;
```

Comment:
Please check @andela-dbamidele's PR, I explained how to fix css without using `!important`